### ISSUE TITLE
Hyperparameters selection using corrupted validation set

### ISSUE THREAD

**Issue by xwuShirley** [shengliu66/ELR#1]
Dear authors,

Thank you for the open-source code.
I have a question regarding F.4 Hyperparameters selection in the appendix. You split the corrupted data into train and validation sets, and then select the hyper-parameter from the validation set. I wonder what is your criterion to select that hyper-para. Since the validation set is corrupted, it would not be  optimal by choosing those best validation accuracy. Maybe the worse validation accuracy? 

This perhaps is a general question to label noise. 
Thank you very much.

**Comment by shengliu66** [comment]
Hi!

It is a great question! When CNN is trained on noisy labels, the model will "memorize" the wrong labels, but for an independent validation set with label noise, it is not necessarily true that the model will fit the wrong labels in the validation set as it never saw those examples while training. What we observed is that the best model in the noisy validation set will also perform well in the clean test set (because it also means the model performs well on the cleanly labeled part), and perhaps more importantly, it is a much more practical solution in real-life noisy label problems.
